### DISCUSSION TITLE
Structuring of the Data Management Section

### DISCUSSION THREAD

**Discussion by Selkubi** [TU-Delft-DCC/TU-Delft-DCC.github.io#180] [Archive]
I was just thinking of a more structured way of presenting the FAIR data, including some information on data formats and versioning and came across this poster on the library wall. 

Do you think it makes sense to mention these steps as part of the FAIR data?

<img src="https://github.com/user-attachments/assets/2a831836-92ea-4b47-90fc-92309689b18c" width=50% height=40%>

**Comment by edvinskis** [comment]
The entire *Data Management* section has now been updated. Thank you again for your contributions and for starting this *Discussions* post, which helped us structure that section. I'll close this discussion.
